crates/web5_cli/src/dids/create.rs

```diff
@@ -63,8 +68,60 @@ impl Commands {
 
                 print_portable_did(portable_did, no_indent, json_escape);
             }
-            Commands::Web { domain: _ } => {
-                println!("🚧 not currently supported 🚧");
+            Commands::Web {
+                domain,
+                no_indent,
+                json_escape,
+            } => {
+                let private_jwk = Ed25519Generator::generate();
+                let mut public_jwk = private_jwk.clone();
+                public_jwk.d = None;
+
+                let valid_url = if domain.starts_with("http://") || domain.starts_with("https://") {
```

Comment:
this is really amazing code @diehuxx 👏 

so good, in fact, I'm going to move it inside the `DidWeb::new()` function, so that we enable this DX not just for the CLI experience but also for application-level use cases. I feel confident exposing this code such that we can expose to developers, "send in any endpoint and we'll create a did:web instance out of it"

also I spy a slight bug in the case wherein both a `.well-known` and `did.json` are passed in, like in [Example 4 here](https://w3c-ccg.github.io/did-method-web/#create-register)